conu/docker/core.py

```diff
@@ -147,3 +148,22 @@ def copy_to(self, src, dest):
     def copy_from(self, src, dest):
         self.start()
         run_cmd("docker cp %s:%s %s" % (self.tag, src, dest))
+
+    def check_files_exist(self, files):
```

Comment:
I don't follow why it accepts a list of files and then spawns a new `ls` process for every file: users can easily do a loop like this in their tests, or this function could accepts just a single file, or it could accept `*files` and then just check `ls %s' % " ".join(files)`.